docs/explanations/architecture.rst

```diff
@@ -6,7 +6,18 @@ This page describes the architecture of PostgREST.
 Bird's Eye View
 ===============
 
-.. image:: ../_static/arch.png
+  .. container:: img-dark
+
+    .. raw:: html
+
+      <object width="100%" data="../_static/arch-dark.svg" type="image/svg+xml"></object>
```

Comment:
Using `.. image` does not print an interactive image. This is a workaround I found in https://github.com/sphinx-doc/sphinx/issues/2240#issuecomment-187366626.

Reply:
How about adding that link a a comment here? So we have the reference